dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -743,13 +750,14 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
       in_data_.resize(samples_hw_batched_.size());
       in_lengths_.resize(samples_hw_batched_.size());
       nvjpeg_destinations_.resize(samples_hw_batched_.size());
+      nvjpeg_params_.resize(samples_hw_batched_.size());
       memset(nvjpeg_destinations_.data(), 0, nvjpeg_destinations_.size() * sizeof(nvjpegImage_t));
+      memset(nvjpeg_params_.data(), 0, nvjpeg_params_.size() * sizeof(nvjpegDecodeParams_t));
```

Comment:
```suggestion
      nvjpeg_destinations_.clear();
      nvjpeg_destinations_.resize(samples_hw_batched_.size());
      nvjpeg_params_.clear();
      nvjpeg_params_.resize(samples_hw_batched_.size());
```
This should default-fill the vectors, even for POD types and it stays correct if that type is replaced with something not as trivial. Also, it's shorter and potentially avoids a copy in `resize` on reallocation.